api/user-story/models/user-story.js

```diff
@@ -23,6 +23,23 @@ module.exports = {
           })
         }
       }
+
+      catch(e) {}
+    },
+    async beforeUpdate(params,data) {
+      try {
+           const story=await strapi.query('user-story').findOne({id: params.id})
+           const userId= data.followers[data.followers.length-1]
+           if(story?.followers?.length < data?.followers?.length && story.author.id !== userId) {
+           const user= await strapi.query('user', 'users-permissions').findOne({id: userId})
+           await strapi.services['user-story-notification'].create({
+            message: `${user.username} voted up your story`,
+            users: [story.author],
+            date: new Date(),
+            link: `story/${story.id}`
+           })
+          }
```

Comment:
From line 31 to line 41 the indentation is not correct. I suggest that you configure your IDE to automatically format your indentation with 2 spaces. Visual Code Studio has this option for Formatting.

It's a very good practice to have consistent indentation and formatting, and we're very picky about it at EOS, so let's make sure this file is set correctly

Reply:
Done.